### PR TITLE
Update , delete and CSS changes

### PR DESCRIPTION
**Pull request by kiranrsp**
Update , delete and CSS changes

### CODE REVIEW

File: main_app/templates/home.html [hunk 1]
```diff
@@ -3,36 +3,70 @@
...
+                    <td>{{ item.description }}</td>
+                    <td>{{ item.quantity_current }}</td>
+                    <td>{{ item.quantity_max }}</td>
+                    <td>
+                        <a class="btn btn btn-info" href="{% url 'item_update' item.id %}">
+                          <i class="fa-solid fa-eye fa-lg"></i>
+                        </a>
+                      </td>
+                    <td>
+                      <a class="btn btn-warning me-1" href="{% url 'item_update' item.id %}">
```
Comment (MartyEternal) [inline main_app/templates/home.html:46]: What's going on here with the warning button? I didn't see any extra button in the screenshot you showed us earlier today.

File: main_app/templates/base.html [hunk 2]
```diff
@@ -1,61 +1,73 @@
...
+          <ul class="navbar-nav me-auto">
+            <li class="nav-item dropdown">
+                <a class="nav-link dropdown-toggle" data-bs-toggle="dropdown" href="#" role="button" aria-haspopup="true" aria-expanded="false">Features</a>
+                <div class="dropdown-menu">
+                  <a class="dropdown-item" href="{% url 'about' %}">About</a>
+                  <a class="dropdown-item" href="#">Contact</a>
+                  <a class="dropdown-item" href="#">Pricing</a>
+                </div>
+              </li>
+              {% if user.is_authenticated %}
```
Comment (MartyEternal) [inline main_app/templates/base.html:28]: Having everything except the "Features" dropdown menu on the nav bar hidden for users who aren't logged on seems a little lonely. Thoughts?

File: main_app/templates/base.html [hunk 3]
```diff
@@ -1,61 +1,73 @@
...
-  </footer>
-</body>
+    <div class="container-fluid">
+      {% block content %}
+      {% endblock %}
+    </div>
+    <footer class="mt-auto mb-4">
+      <div class="text-center">
+        <span>
+          Copyright &copy; <script>document.write(new Date().getFullYear())</script>&nbsp;&nbsp;StockSense: Inventory Management System
```
Comment (MartyEternal) [inline main_app/templates/base.html:66]: This footer will keep changing to the current year, right? Shouldn't it just be the year we developed the app?

File: main_app/templates/base.html [hunk 4]
```diff
@@ -1,61 +1,73 @@
...
+        </span>
+      </div>
+    </footer>
+    <script src="https://cdn.jsdelivr.net/npm/bootstrap@5.0.2/dist/js/bootstrap.bundle.min.js"></script>
```
Comment (MartyEternal) [inline main_app/templates/base.html:70]: Confirming this JS is functional for the dropdown only or anything else implemented so far?

File: main_app/templates/home.html [hunk 5]
```diff
@@ -3,36 +3,70 @@
 {% block content %}
 
 {% if user.is_authenticated %}
-<div class="col-md-6">
-  <h1>Welcome to IVM</h1>
```
Comment (MartyEternal) [inline main_app/templates/home.html:7]: Good decision to take away the h1 here. The header already has the app's name on it on the nav bar, so we can just keep the focus of the body on the actual content. Good job.

File: main_app/templates/home.html [hunk 6]
```diff
@@ -3,36 +3,70 @@
...
+      <div class="card-body">
+        <p class="card-text">
+          <div class="table-responsive ">
+            <table class="table table-hover">
+              <thead>
+                <tr>
+                    <th scope="col">ID</th>
+                    <th scope="col">Name</th>
+                    <th scope="col">Type</th>
+                    <th scope="col">Description</th>
```
Comment (MartyEternal) [inline main_app/templates/home.html:23]: Was thinking of having the order of these columns be a little different.
ID | Name | Type | Count | Stock Status | Edit | Delete

Stock status is a conditional to show the level of the quantity_current compared to the quantity_min to make sure that the users can quickly see what needs to be restocked.

File: main_app/templates/item/detail.html [hunk 7]
```diff
@@ -14,10 +14,7 @@ <h1>Item Details</h1>
         <p>There are only <span style="color: red">{{ item.quantity_current }}</span> remaining. Please restock.</p>
         {% endif %}
     </div>
-    <div class="card-action">
-        <a href="{% url 'inventory_update' item.id %}">Edit</a>
```
Comment (MartyEternal) [inline main_app/templates/item/detail.html]: Are we sure we only want the options to edit/delete on the home page for the list view?

File: main_app/templates/base.html (hunk 2 above)
Comment (jackson-07) [reply main_app/templates/base.html:28]: Yeah I tend to agree, what features were we hoping to have here and are they clickable links?

**Review by MartyEternal — COMMENTED** [review]
Have left single comments here and there. Happy with most of the work here. Waiting for all to agree to push to main branch.

File: main_app/templates/home.html (hunk 5 above)
Comment (jackson-07) [reply main_app/templates/home.html:7]: Agreed, Nav Bar does the job for this, good job!

**Review by MartyEternal — APPROVED** [review]
LGTM

**Review by MartyEternal — APPROVED** [review]
LGTM

**Pull request merged**